### PR TITLE
Terraform v0.14 update

### PR DESCRIPTION
**Pull request by robmorgan**
This PR updates the repo to support Terraform 0.14.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,11 +7,12 @@ env: &env
   environment:
     GRUNTWORK_INSTALLER_VERSION: v0.0.30
     TERRATEST_LOG_PARSER_VERSION: v0.30.4
-    MODULE_CI_VERSION: v0.29.1
-    TERRAFORM_VERSION: 0.13.5
+    MODULE_CI_VERSION: v0.31.0
```
Comment (marinalimeira) [inline .circleci/config.yml:10]: Update module CI to at least v0.32.0

Comment (robmorgan) [inline .circleci/config.yml:10]:
```suggestion
    MODULE_CI_VERSION: v0.33.2
```

Comment (robmorgan) [reply .circleci/config.yml]: Changed to v0.33.2

**Review by marinalimeira — APPROVED** [review]
LGTM

**Pull request merged**